docs/source/raft_ann_benchmarks.md

```diff
@@ -60,7 +155,6 @@ Configuration files already exist for the following list of the million-scale da
 - `fashion-mnist-784-euclidean`
 - `glove-50-angular`
 - `glove-100-angular`
-- `lastfm-65-angular`
```

Comment:
Is this one not working?

Reply:
nope, at least not downloaded from the original location as we had it. We can add it back in the scripts and then add it back here as a follow up